CONTRIBUTORS.yaml

```diff
@@ -235,6 +235,10 @@ davidchristiany:
     name: David Christiany
     joined: 2019-04
 
+davebx:
```

Comment:
@davebx please confirm this is ok, I didn't add your last name since it isn't exposed on GH. But if you'd like to share more information here in your "contributor profile", please feel free to do so!

Reply:
Full name is fine, though I do appreciate the ping.